schema_utils.py

```diff
@@ -229,47 +262,120 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to compare with min_value.
+            min_value: int. Minimum value of elements.
+
+        Returns:
+            bool. True if the number of elements is greater than or equal
```

Comment:
bool. Whether the number of elements in `obj` is greater than or equal to min_value.